core/src/main/scala/com/databricks/labs/remorph/preprocessors/Processor.scala

```diff
@@ -0,0 +1,10 @@
+package com.databricks.labs.remorph.preprocessors
+
+import com.databricks.labs.remorph._
+import org.antlr.v4.runtime.{CharStream, Lexer}
+
+trait Processor extends TransformationConstructors {
```

Comment:
```suggestion
trait Processor {
```
Let the descendants of `Processor` decide whether they need to mix `TransformationConstructors`.

Reply:
I see. Now I have to go back and see why I thought that was needed ;)